File: problemas/iniciante/1095/1095.exs
defmodule Modulo do
  def imprime(i, j) do
    IO.puts "I=#{i} J=#{j}"
    if j == 0 do
      :ok
    else
      imprime(i + 3, j - 5)
    end
  end
end

Modulo.imprime(1, 60)

File: problemas/iniciante/1095/1095.hs
import Text.Printf

imprime :: Int -> Int -> IO ()
imprime i 0 = printf "I=%d J=0\n" i
imprime i j = do
  printf "I=%d J=%d\n" i j
  imprime (i + 3) (j - 5)

main :: IO ()
main = imprime 1 60